hedyorg/hedy
# [WEBSITE] Implements dynamic multi-langual logo

**Description**
Implements a dynamic text placeholder for the logo.

**Fixes**
This PR fixes #4080.

**How to test**
Try different languages, verify that is always works. Especially in the case of LTR and RTL.
